Kiosk/App/CardHandler.swift

```diff
@@ -54,10 +54,14 @@ class CardHandler: NSObject, CFTReaderDelegate {
             logger.log("CardReader got a response it cannot handle")
 
 
-            reader.beginSwipeWithMessage(nil);
+            reader.beginSwipe();
         }
     }
 
+    func transactionResult(charge: CFTCharge!, withError error: NSError!) {
```

Comment:
This is a new, required method. Event though we don't do any transactions, we still need to implement it.